Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -466,15 +466,15 @@ - (void)logOut
 #else
 
   if([NYPLBookRegistry sharedRegistry].syncing == YES) {
-    [self presentViewController:[NYPLAlertController
+    [self presentViewController:[NYPLAlertUtils
                                  alertWithTitle:@"SettingsAccountViewControllerCannotLogOutTitle"
                                  message:@"SettingsAccountViewControllerCannotLogOutMessage"]
                        animated:YES
                      completion:nil];
   } else {
-    [[NYPLMyBooksDownloadCenter sharedDownloadCenter] reset:self.selectedAccountType];
-    [[NYPLBookRegistry sharedRegistry] reset:self.selectedAccountType];
-    [[NYPLAccount sharedAccount:self.selectedAccountType] removeAll];
+    [[NYPLMyBooksDownloadCenter sharedDownloadCenter] reset:self.selectedNYPLAccount.userID];
```

Comment:
what are these changes for? they seem unrelated to this PR

Reply:
Without this changes, project didn't build.
This hunk of code lies in else part of #if FEATURE_DRM_CONNECTOR check (which means it wasn't compiled during development with Adobe).
Variable self.selectedAccountType no longer exists.I noticed that a piece of code inside FEATURE_DRM_CONNECTOR part uses self.selectedNYPLAccount.userID as a parameter in those methods.

Reply:
oh I see, gotcha. BTW selectedNYPLAccount has been renamed recently, make sure to rebase on top of develop. 

But I also wonder if you should use `self.selectedAccountId` which refers to the library currently selected. Judging from the `reset:` method in NYPLMyBooksDownloadCenter that's probably what you want. 

Some background here: `account` as terminology is a bit confusing. There's an Account class that essentially identifies a library. Then there's a `NYPLAccount` class that means "User account". @ErnestFan is rewriting that in swift and renaming it to NYPLUserAccount for more clarity.

Reply:
@ettore of course you were right, I changed `self.selectedNYPLAccount.userID` to `self.selectedAccountId`